tests/test-cache.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Class Cache_Tests
+ *
+ * @package WP_Irving
+ */
+
+/**
+ * Tests for integration with WPCOM Legacy Redirector.
+ */
+class Cache_Tests extends WP_UnitTestCase {
+
+	/**
+	 * Helpers class instance.
+	 *
+	 * \WP_Irving_Test_Helpers
+	 */
+	static $helpers;
+
+	/**
+	 * Helpers class instance.
+	 *
+	 * \WP_Irving\Cache
+	 */
+	static $cache;
+
+	/**
+	 * Components endpoint instance.
+	 *
+	 * \WP_Irving\REST_API\Components_Endpoint
+	 */
+	static $components_endpoint;
+
+	/**
+	 * Test suite setup.
+	 */
+	public static function setUpBeforeClass() {
+		global $wp_rewrite;
+		$wp_rewrite->set_permalink_structure( '/%year%/%monthnum%/%day%/%postname%/' );
+
+		self::$helpers = new \WP_Irving\Test_Helpers();
+		self::$cache   = \WP_Irving\Cache::instance();
+	}
+
+	/**
+	 * Test post purge urls.
+	 */
+	public function test_get_post_purge_urls() {
+		$current_user = $this->factory->user->create_and_get(
+			[
+				'user_login'  => 'alley',
+			]
+		);
+		$current_post = $this->factory->post->create_and_get(
+			[
+				'post_title'  => rand_str(),
+				'post_date'   => '2020-01-01 00:00:00',
+				'post_author' => $current_user->ID,
+			]
+		);
+		$current_term = $this->factory->term->create_and_get(
+			[
+				'name'     => 'Test',
+				'taxonomy' => 'post_tag',
+			]
+		);
+		// print_r( $current_term );
+		$this->factory->term->add_post_terms( $current_post->ID, [ $current_term->slug ], 'post_tag' );
+
+		$this->assertEquals(
+			self::$cache->get_post_purge_urls( $current_post->ID ),
+			[
+				'http://example.org/2020/01/01/' . $current_post->post_title . '/',
+				'http://example.org/',
+				'http://example.org/?cat=1',
```

Comment:
I just cannot seem to figure out why term permalinks aren't structured properly in these tests. They should be `/category/slug` not `?cat=id`

Reply:
If you put `update_option( 'permalink_structure', '/%year%/%monthnum%/%day%/%postname%/' );` in the `_manually_load_plugin` function in `bootstrap.php` that'll fix the issue